phase1/gce/configure-vm-kubeadm.sh

```diff
@@ -27,7 +27,7 @@ fi
 
 case "${ROLE}" in
   "master")
-    kubeadm init --token "${TOKEN}" --apiserver-bind-port 443 --skip-preflight-checks --apiserver-advertise-address "$(get_metadata "k8s-advertise-addresses")" --kubernetes-version $KUBERNETES_VERSION
+    kubeadm init --token "${TOKEN}" --pod-network-cidr 10.244.0.0/16 --apiserver-bind-port 443 --skip-preflight-checks --apiserver-advertise-address "$(get_metadata "k8s-advertise-addresses")" --kubernetes-version $KUBERNETES_VERSION
```

Comment:
Rather than hardcoding the new flag (which not everyone is going to want set, let alone to this value), can you plumb it through so that it comes from `.config` instead? Or, since flannel requires this specific value, you can make it so that enabling flannel forces this flag, but disabling flannel omits the cidr flag entirely.